src/coreclr/pal/src/eventprovider/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(EVENT_MANIFEST ${VM_DIR}/ClrEtwAll.man)
 
-if(CLR_CMAKE_HOST_LINUX)
+# TODO: https://github.com/dotnet/runtime/issues/111491
```

Comment:
LTTNG is not supported by Mono (https://github.com/dotnet/runtime/issues/65181). I do not think there is anything to do. We do need to support anything beyond what Mono supports.

(Also, LTTNG is not a tracing tech we are betting on going forward in general.)

Reply:
Removed TODO